pydantic/fields.py

```diff
@@ -184,6 +184,10 @@ def _singleton_schema(self, by_alias):
                     'type': 'any_of',
                     'types': [sf.type_schema(by_alias) for sf in self.sub_fields]
                 }
+        elif self.type_ is Any:
+            return {
```

Comment:
wait, shouldn't this just be `return 'any'` same as `return display_as_type(self.type_)`?

Reply:
agree